provider/storage_benchmark.py

```diff
@@ -337,20 +337,29 @@ def run(self, cmd_options='-a', timeout=1800):
 
 class FioLinuxCfg(object):
     def __init__(self, params, session):
-        fio_pkg = params.get("fio_pkg", 'fio-3.13-48-ga819.tar.bz2')
-        host_path = os.path.join(data_dir.get_deps_dir(), 'fio', fio_pkg)
-        self.download_path = os.path.join('/home', fio_pkg)
-        self.fio_inst = os.path.join('/home', 'fio_inst')
-        self.fio_path = '%s/bin/fio' % self.fio_inst
-        scp_benckmark = attrgetter('scp_benckmark')
-        unpack_file = attrgetter('unpack_file')
-        install_timeout = params.get_numeric('fio_install_timeout', 300)
-        install = attrgetter('install')
-        self.setups = {scp_benckmark: (params.get('username'), params.get('password'),
-                                       host_path, self.download_path),
-                       unpack_file: (TAR_UNPACK, self.download_path, self.fio_inst),
-                       install: (self.fio_inst, self.fio_inst, install_timeout)}
-        self.setup_orders = (scp_benckmark, unpack_file, install)
+        fio_resources = params.get("fio_resources", 'fio-3.13-48-ga819.tar.bz2')
+        for fio_resource in fio_resources.split(','):
```

Comment:
Why need multiple fio resources?

Reply:
More flexible, one more chance if the high priority one failed, and no effect if user want to use one resource only.

Reply:
@YongxueHong As we discussed, do not extend the requirement to support multiple choice.
User can choose either 'distro' or 'pkg'.
Patch updated, could you help re-review ? Thankse